docs/reference/esql/functions/types/e.asciidoc

```diff
@@ -1,5 +1,5 @@
 [%header.monospaced.styled,format=dsv,separator=|]
 |===
 result
-integer | double
```

Comment:
thought: Hm, if we still emit this table for constants like `e`, at all, then I wonder if it wouldn't be better to still place the result type here, i.e.
`double`
instead of an empty line.

Then, we could consistently add the type table to both constants and functions.

Reply:
I hadn't even looked at this table for the constant functions. I had figured we'd build the file because it didn't hurt but that we wouldn't use it. I suppose it's pretty simple to make it useful though.